PTGLdynamics/codes/plotting-scripts/PyMolHeatmapVisualisation.py

```diff
@@ -172,6 +178,27 @@ def sorted_nicely( l ):
                        default = '',
                        help = 'specify a path to your output files. Otherwise the current folder is used.')
 
+cl_parser.add_argument('--divide-by-chainlength',
+                       metavar = 'divide-by-chainlength',
+                       default = '',
+                       help = 'specify a path to a "number_of_residues_in_each_chain" csv file to divide the change of each chain by its chain length.')
+
+# Args for the computation of the csv file containing the number of residues in each chain. 
+cl_parser.add_argument('--pdb-or-cif-file',
```

Comment:
I know that we sometimes refer to legacy pdb files just as pdb files because of the ending, but both legacy pdf files and mmCIF are pdb files. Therefore, I suggest to rename the option to `--pdb-file`.